file.txt

```diff
@@ -0,0 +1,3 @@
+This is some
+ content for
+ the file.
```

Comment:
Do not push redundat files into Pull reqest : )

Reply:
XD